ENIGMAsystem/SHELL/Universal_System/roomsystem.cpp

```diff
@@ -648,14 +612,6 @@ int room_add()
   rm->createcode = NULL;
   rm->precreatecode = NULL;
 
-  enigma::inst *newinst = new enigma::inst[1];
-  rm->instances = newinst;
-  rm->instancecount = 0;
-
-  enigma::tile *newtiles = new enigma::tile[1];
```

Comment:
This should have been replaced with a resize. The size of 1 there is for the dummy room instance I believe.